data/templates/telegraf/telegraf.j2

```diff
@@ -45,7 +45,11 @@
 ### Prometheus ###
 [[outputs.prometheus_client]]
   ## Address to listen on
+{%     if prometheus_client.listen_address | is_ipv6 %}
+  listen = "{{ prometheus_client.listen_address | bracketize_ipv6 }}:{{ prometheus_client.port }}"
+{%     else %}
   listen = "{{ prometheus_client.listen_address if prometheus_client.listen_address is vyos_defined else '' }}:{{ prometheus_client.port }}"
+{%     endif %}
```

Comment:
I guess it should be 
```
listen = "{{ prometheus_client.listen_address | join() | bracketize_ipv6 if prometheus_client.listen_address is vyos_defined else '' }}:{{ prometheus_client.port }}
```

Reply:
That is much cleaner, il try and update and test it later this evening.